android/build.gradle

```diff
@@ -39,7 +39,7 @@ dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     compile 'com.android.support:appcompat-v7:25.3.1'
     compile 'com.android.support:design:25.3.1'
-    compile "com.twilio:video-android:1.0.1"
+    compile "com.twilio:video-android:1.3.8"
```

Comment:
Ignorant on Android here, is it possible to add version constraints instead of pinned versions here ?

Reply:
not that i could determine.

Reply:
You can do 1.+

But android dependencies are often a pain so I usually just leave them pinned.